unix/vncpasswd/vncpasswd.cxx

```diff
@@ -37,6 +37,15 @@
 
 #include <termios.h>
 
+#include <pwquality.h>
+#include <locale.h>
+#include <libintl.h>
```

Comment:
Let's try to use the same convenience wrappers that we have for vncviewer. Please refer to the files `i18n.h` and `gettext.h` there.

Reply:
Okay, I'll try to modify it.